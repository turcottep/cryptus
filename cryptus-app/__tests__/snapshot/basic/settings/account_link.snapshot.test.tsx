import React from "react";
import renderer from "react-test-renderer";
import "@testing-library/jest-dom/extend-expect";

import AccountLink from "../../../../components/basic/settings/account_link/account_link";

describe("<AccountLink />", () => {
  it("should display properly", async () => {
    const mockProps = {
      username: "TristanIsTesting",
    };

    const tree = renderer.create(<AccountLink {...mockProps} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
