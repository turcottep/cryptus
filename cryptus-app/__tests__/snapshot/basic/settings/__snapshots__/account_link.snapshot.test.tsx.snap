// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AccountLink /> should display properly 1`] = `
<div>
  <div>
    Account link
  </div>
  <div
    onClick={[Function]}
  >
    publicwallet.app/TristanIsTesting
    <div>
      <img
        src="/icons/copy_icon.png"
      />
    </div>
  </div>
</div>
`;
